docs/scos/dev/front-end-development/202307.0/oryx/building-components/component-definition.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Oryx components definition"
+description: Components are registered in an Oryx application by a definition file
+last_updated: Sept 19, 2023
+template: concept-topic-template
+---
+
+Oryx components can be integrated in different ways. They can be listed in [pages](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-pages/oryx-pages.html) and [compositions](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-pages/oryx-compositions.html) and used inside other components or content integrated from a third-party CMS.
```

Comment:
Do you think `listed` matches the meaning here?

Reply:
A page/composition has a list of components, but agreed that "uses" is better here

Reply:
rephrased